client/src/hooks/useOrganizationBests.js

```diff
@@ -75,6 +81,27 @@ export default function useOrganizationBests() {
         );
```

Comment:
`neighborhoodId` currently doesn't exist on the `filters` object. We can remove this condition, and if we want to implement this in the future, we can revisit this.